proposals/route-blinding.md

```diff
@@ -261,7 +261,9 @@ total `cltv_expiry_delta`. This yields the following values:
 * `route_cltv_expiry_delta`: 300
 
 Let's assume the current block height is 1000. Alice wants the route to be used in the next 200
-blocks, so she sets `max_cltv_expiry = 1200` and adds `cltv_expiry_delta` for each hop. Alice then
+blocks, meaning that the `max_cltv_expiry` she will communicate to the payer will be 1200. She 
```

Comment:
What is meant by "the `max_cltv_expiry` she will communicate to the payer"? Does that mean through the invoice's `payinfo.cltv_expiry_delta`? If so, why is that a delta while `max_cltv_expiry` is absolute? 🤔 

AFAICT, LDK doesn't include `min_final_cltv_expiry_delta` in the encrypted payload calculation, but *does* include it in what Alice communicates to the payer (i.e., in `payinfo.cltv_expiry_delta`), which seems like the opposite of what is stated here.

@valentinewallace Could confirm / provide some insight here? You have a better understanding of this than I do. Background: https://github.com/lightning/bolts/issues/1174#issue-2371364610

Reply:
^ That's all accurate from my understanding. 

>What is meant by "the max_cltv_expiry she will communicate to the payer"?

I tried to clarify this in the past via https://github.com/lightning/bolts/pull/1069, i.e the `max_cltv_expiry` should be conveyed via the invoice absolute expiry. But LDK doesn't do this lol, we just choose a very buffered absolute CLTV expiry.

Reply:
Is my understanding right, though, that LDK doesn't include `min_final_cltv_expiry_delta` in the encrypted data calculation? That seems at odds with the change in the first commit.

Reply:
I believe we don't need to *explicitly* include `min_final_cltv_expiry_delta` in our encrypted data calculation because we add so much buffer as mentioned above. Because senders only see the invoice expiry (2 hours), the final expiry that they set shouldn't come close to violating our min final delta. Does that make sense to you?